code/modules/core_implant/cruciform/rituals/path.dm

```diff
@@ -63,9 +63,9 @@
 	desc = "Heal every person who can see and hear you for a fair amount, even if they do not have a cruciform. This prayer uses alot of power and requires five minutes to pass before it can be used again."
```

Comment:
```suggestion
	desc = "Heal every person who can see and hear you for a fair amount, even if they do not have a cruciform. This prayer uses alot of power and requires 15 minutes to pass before it can be used again."
```